contents/docs/ai-engineering/llm-insights.md

```diff
@@ -19,11 +19,11 @@ Currently, we support integrations for the following platforms:
 
 - [Langfuse](https://langfuse.com) ([instructions](/docs/ai-engineering/langfuse-posthog))
 - [Helicone](https://www.helicone.ai/) ([instructions](/docs/ai-engineering/helicone-posthog))
-
+- [Traceloop](https://www.traceloop.com/) ([instructions](/docs/ai-engineering/traceloop-posthog))
```

Comment:
I think we can just link to the instructions here instead of both the app and instructions. Each of the instructions has an app link and they are all a good search away, I think it makes it a lot cleaner.